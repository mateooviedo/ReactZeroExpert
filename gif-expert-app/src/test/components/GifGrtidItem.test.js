import React from 'react';
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('GifGridItem component', () => {
	const title = 'un titulo';
	const url = 'url image';
	const wrapper = shallow(<GifGridItem title={title} url={url} />);

	test('Should show the component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});

	test('Should it has a paragraph in the title', () => {
		const p = wrapper.find('p');
		expect(p.text()).toBe(title);
	});
});
