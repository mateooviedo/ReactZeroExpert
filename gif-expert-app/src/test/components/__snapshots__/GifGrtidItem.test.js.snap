// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifGridItem component Should show the component correctly 1`] = `
<div
  className="card animate__animated animate__fadeInDown"
>
  <img
    alt="un titulo"
    src="url image"
  />
  <p>
    un titulo
  </p>
</div>
`;
